### PR TITLE
Tweaks

### PR DESCRIPTION
**Pull request by michield**
## Description
Various small tweaks

### CODE REVIEW

**Comment by michield** [comment]
Hmm, the CI is failing. Needs some work

File: public_html/lists/admin/index.php [hunk 1]
```diff
@@ -65,7 +65,7 @@ function mb_strtolower($string)
 } elseif (isset($cline['c']) && is_file($cline['c'])) {
     $configfile = $cline['c'];
 } elseif (is_file(dirname(__FILE__).'/../config/config.php')) {
-    $configfile = '../config/config.php';
+    $configfile = dirname(__FILE__).'/../config/config.php';
```
Comment (bramley) [inline public_html/lists/admin/index.php:68]: As the code is being changed it might be worth using `__DIR__` instead of `dirname(__FILE__)`

**Review by bramley — COMMENTED** [review]
Mostly looks fine but a few suggested changes and one question.

File: public_html/lists/admin/init.php [hunk 2]
```diff
@@ -158,9 +153,15 @@
 
 if (empty($GLOBALS['language_module'])) {
     $GLOBALS['language_module'] = 'english.inc';
+    if (isset($language_module)) {
+      $GLOBALS['language_module'] = $language_module;
```
[the remark this answers is not in the document]
Comment (michield) [reply public_html/lists/admin/init.php:157]: I think I got an "index doesn't exist" error at some point. Not sure. You're right, but it won't matter so much to leave it.

File: public_html/lists/config/config.php [hunk 3]
```diff
@@ -32,7 +32,7 @@
 //#
 //#     define("PHPMAILERHOST",'smtp.mydomain.com');
 
-define('PHPMAILERHOST', 'localhost');
+define('PHPMAILERHOST', '');
```
[the remark this answers is not in the document]
Comment (michield) [reply public_html/lists/config/config.php:35]: Yes, I'll update that.

File: public_html/lists/admin/admin.php [hunk 4]
```diff
@@ -285,7 +285,11 @@
     if ($id) {
         $val_req = Sql_Fetch_Row_Query("select value from {$tables['admin_attribute']}
       where adminid = $id and adminattributeid = $row[id]");
-        $row['value'] = $val_req[0];
+        if (!empty($val_req[0])) {
+          $row['value'] = $val_req[0];
```
[the remark this answers is not in the document]
Comment (michield) [reply public_html/lists/admin/admin.php:289]: I'll change `!empt`y to `isset`, and that will handle 0 values.

File: public_html/lists/admin/defaultsystemtemplate.php [hunk 5]
```diff
@@ -404,7 +404,7 @@
         , $title
     ));
     
-    if ($exists[0]) {
+    if (!empty($exists[0])) {
```
[the remark this answers is not in the document]
Comment (michield) [reply public_html/lists/admin/defaultsystemtemplate.php:407]: It's kind of the same. All I tried to to was avoid a warning.

**Comment by michield** [comment]
Suggestions applied.

**Review by aulona1 — APPROVED** [review]

**Pull request merged**